src/main/public/webchat/javascript/webchart-init.js

```diff
@@ -17,10 +17,10 @@ xhttp.onreadystatechange = function () {
       chatDownAction: 'showMessage',
       chatLinkFocusable: false,
       textChatDown: 'The web chat service is temporarily unavailable, please try again later.',
-      textChatClosed: 'Web chat is now closed. Come back Monday to Friday 9am to 5pm, or contact us by phone or email.',
+      textChatClosed: 'Web chat is now closed. Come back Monday to Friday 9am to 5pm. Or contact us using one of the ways below.',
```

Comment:
not sure you should start a sentence with **Or**

Reply:
very true